lib/chef/resource/aws_rds_option_group.rb

```diff
@@ -0,0 +1,25 @@
+require 'chef/resource/aws_resource'
+require 'chef/provisioning/aws_driver'
+
+class Chef::Resource::AwsRdsOptionGroup < Chef::Resource::AwsResource
+  self.resource_name = 'aws_rds_option_group'
```

Comment:
This should be updated to use the `aws_sdk_type` (even though the option groups don't have a model)